spec/core/ics-002-client-semantics/README.md

```diff
@@ -71,8 +76,8 @@ could be provided as executable WASM functions when the client instance is creat
 
 * `ConsensusState` is an opaque type representing the state of a validity predicate.
   `ConsensusState` must be able to verify state updates agreed upon by the associated consensus algorithm.
-  It must also be serialisable in a canonical fashion so that third parties, such as counterparty machines,
-  can check that a particular machine has stored a particular `ConsensusState`. It must finally be
+  It must also be serialisable in a canonical fashion so that third parties, such as counterparty state machines,
```

Comment:
for counterparty the term remote was used before. Perhaps we should fix one, and use it everywhere?

Reply:
```suggestion
  It must also be serialisable in a canonical fashion so that third parties, such as remote state machines,
```